morpheus/_lib/include/morpheus/stages/add_scores_stage_base.hpp

```diff
@@ -68,6 +73,8 @@ class AddScoresStageBase
      */
     AddScoresStageBase(std::map<std::size_t, std::string> idx2label, std::optional<float> threshold);
 
+    std::shared_ptr<OutputT> pre_process_batch(std::shared_ptr<InputT> x);
+
```

Comment:
This class already as the `on_data` method. We dont need to add a new method here. Just make `on_data` public so it can be used in tests